scripts/jenkins-yoctobuild-build.sh

```diff
@@ -499,6 +501,26 @@ then
     PATH="$OLD_PATH"
 fi
 
+if [ "$BUILD_RPI3" = "true" ]
+then
+    cd "$WORKSPACE"
+    source oe-init-build-env build-rpi3
+    prepare_build_config rpi3
+    disable_mender_service
+
+    bitbake core-image-full-cmdline
+
+    OLD_PATH="$PATH"
```

Comment:
Don't forget `PATH="$OLD_PATH"` at the end, or the later steps will be messed up.